pallets/polkadex-ido/src/test.rs

```diff
@@ -679,17 +679,21 @@ fn test_show_interest_in_round_randomized_participants() {
         let round_id = <InfoProjectTeam<Test>>::get(ALICE.clone()).unwrap();
         assert!(PolkadexIdo::approve_ido_round(Origin::signed(1_u64), round_id).is_ok());
         let investors: Vec<(u64, Balance)> =
-            vec![(4u64, 200), (2u64, 200), (5u64, 200), (6u64, 300)];
+            vec![(4u64, 200), (2u64, 200), (5u64, 200)];
 
         let funding_round: FundingRound<Test> = <WhitelistInfoFundingRound<Test>>::get(round_id).unwrap();
 
         system::Pallet::<Test>::set_block_number(funding_round.start_block);
 
-        for (investor_address, amount) in investors {
+        for (investor_address, amount) in investors.clone() {
             assert_eq!(
                 PolkadexIdo::register_investor(Origin::signed(investor_address)),
                 Ok(())
             );
+        }
+
+        for (investor_address, amount) in investors {
+            dbg!(&amount);
```

Comment:
I assume that this line was added for debugging in process of test implementation. Please remove it if so.